Scripts/DowloadPortalItemData.swift

```diff
@@ -161,10 +162,10 @@ func downloadFile(at sourceURL: URL, to downloadDirectory: URL, completion: @esc
                 
                 if isArchive {
                     let extractURL = downloadURL.pathExtension == "zip"
-                        // Uncompresses to directory named after archive.
-                        ? downloadURL.deletingPathExtension()
-                        // Uncompresses to appropriate subdirectory.
-                        : downloadURL.deletingLastPathComponent()
+                    // Uncompresses to directory named after archive.
+                    ? downloadURL.deletingPathExtension()
+                    // Uncompresses to appropriate subdirectory.
+                    : downloadURL.deletingLastPathComponent()
```

Comment:
Inadvertent whitespace changes?

Reply:
This happened when I re-indented the file. Should it not be formatted like this?

Reply:
How the ternary operator is indented is a matter of preference, but it would be better to revert these changes since they are unrelated to the purpose of the PR.